test/test_jit_cuda_fuser.py

```diff
@@ -995,6 +995,80 @@ def test_reduction(self):
                         perm1 = range(len(x))
                         self._reduction_helper(x, axes, torch.float32, "cuda", perm0, perm1, keepdim)
 
+    def _layer_norm_autodiff_helper(self, model, grad, shapes, args):
+        jit_model = torch.jit.script(model)
+
+        eps = np.random.random() * 1e-4
+        use_cudnn = bool(np.random.randint(0, 2))
```

Comment:
Should we just run both?

Reply:
I wouldn't worry too much about it yet. We are currently ignoring the cudnn flag, so does pytorch :)
https://github.com/pytorch/pytorch/blob/3721962c33a64946bd3be8ce8c032c4e7d895d22/aten/src/ATen/native/layer_norm.cpp#L99